src/main/java/com/internet/mixshop/controller/LoginController.java

```diff
@@ -0,0 +1,40 @@
+package com.internet.mixshop.controller;
+
+import com.internet.mixshop.exception.AuthenticationException;
+import com.internet.mixshop.lib.Injector;
+import com.internet.mixshop.model.User;
+import com.internet.mixshop.security.AuthService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.mixshop");
+    private AuthService authService = (AuthService) injector.getInstance(AuthService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/view/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            User user = authService.login(login, password);
+            HttpSession session = req.getSession();
+            session.setAttribute("user_id", user.getId());
```

Comment:
let's make user_id a constant as in filter

Reply:
done